src/entries/popup/components/CommandK/useCommands.tsx

```diff
@@ -229,6 +231,15 @@ export const staticCommandInfo: CommandInfo = {
     symbolSize: 14,
     type: SearchItemType.Shortcut,
   },
+  exportAddressList: {
+    name: 'Export address list to CSV',
```

Comment:
I like the CSV addition. What about "Export Addresses as CSV" instead for capitalization consistency?

Reply:
I realized these strings aren't covered by i18n, so created a backlog ticket for that here: https://linear.app/rainbow/issue/BX-1112/command-k-command-name-i18n